### DISCUSSION TITLE
Text disappears while zooming

### DISCUSSION THREAD

**Discussion by skyrpex** [webpack-contrib/webpack-bundle-analyzer#568] [Ideas]
When zooming in or out the view, the module names disappear for 2s approx. I don't know if it happens for optimization purposes but could that be changed? It's kind of awkward to wait for the filenames to appear again.

Thanks.

**Comment by valscion** [comment]
Not sure if FoamTree supports such a feature: https://carrotsearch.com/foamtree/

Might be impossible to do such a thing. Feel free to experiment :relaxed:
